math/src/main/scala/breeze/linalg/BitVector.scala

```diff
@@ -117,7 +117,7 @@ object BitVector extends BitVectorOps {
       for(i <- from.valuesIterator) fn.visit(i)
     }
 
-    def isTraversableAgain(from: BitVector): Boolean = true
+    override def isTraversableAgain(from: BitVector): Boolean = true
   }
```

Comment:
@dlwh
I don't know that there's an official style guide for breeze, but I'm loosely refactoring off of the databricks guide as I read the code and learn it, is that OK?